src/lib.rs

```diff
@@ -91,78 +91,24 @@
 #![doc(html_logo_url = "https://raw.githubusercontent.com/nvzqz/cargo-emit/assets/logo.svg?sanitize=true")]
 
 #![deny(missing_docs)]
-#![no_std]
 
```

Comment:
Looking back now, I don't see a `println!` macro in `core`. So I guess this crate never was no-std compatible.

Reply:
Yeah, I thought so, too. But wasn't sure if I was missing something subtle.
I added an entry to the 'CHANGELOG' file, too, for transparency.

Reply:
Sure, I'll also bump it to 0.2.0 just in case.